sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -336,7 +338,12 @@ internal class RadarApiClient(
                     RadarSettings.setId(context, user._id)
 
                     if (user.trip == null) {
-                        RadarSettings.setTripOptions(context, null)
+                        // if user was on a trip that ended server side, restore previous tracking options
+                        val tripOptions = RadarSettings.getTripOptions(context)
+                        if (tripOptions != null) {
+                            locationManager.restartPreviousTrackingOptions()
+                            RadarSettings.setTripOptions(context, null)
```

Comment:
@bschoeneweis @corypisano @palmakkar 

After we ship the server changes and turn on the dconfig, we don't need `RadarSettings.set/getTripOptions` anymore, right?  Do we just have it so that `/track` can pass them up for logging and for use during this transition period?

Reply:
tripOptions will still be used in /track if userId is null, and for logging during the transition